_includes/highlights.html

```diff
@@ -9,48 +9,40 @@
-        
-      <div class="usa-media-block__body">
-          <h3 class="usa-graphic-list__heading"><a href="{{ site.baseurl }}/procurement/">Procurement Team</a></h3>
-          <p>Procurement Contracting Officer (PCO), Contracting Officer's Representative (COR), and Accessibility Subject Matter Expert (SME} all play a critical role in choosing digital tools that meet the organization's needs. </p>
+        <div class="usa-media-block__img">{% asset procurement alt="Procurement icon" %}</div>
```

Comment:
I guess not having them inline will make it easier to read. 

I would use `aria-hidden="true"` because there is no real value in the alt text here. It just repeats content and is decorative.

Reply:
Done!